plugins/workload-management/src/main/java/org/opensearch/plugin/wlm/rule/QueryGroupAttribute.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.plugin.wlm.rule;
+
+import org.opensearch.autotagging.Attribute;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Attributes specific to the query group feature.
+ * @opensearch.experimental
+ */
+public enum QueryGroupAttribute implements Attribute {
+    /**
+     * Represents the index_pattern attribute in QueryGroupAttribute
+     */
+    INDEX_PATTERN("index_pattern");
+
```

Comment:
Should this `INDEX_PATTERN` be defined within core and plugins like wlm should just refer to it?

Reply:
This is a valid point, I will move this to the lib folder.